examples/data/mixed_oxide_files/geometry.in

```diff
@@ -0,0 +1,164 @@
+lattice_vector 8.533400 0.000000 0.000000
```

Comment:
Do you need this entire file for the test? I don't mind, but we don't really want to bung up the repository with unnecessary files so it might be nice to think if there is a test that can be done and all you need to do is review the outcome against some number, not compare to a whole file.